src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -846,6 +846,7 @@ public static string GetModulePath(string currentProcessModulePath, string hklmM
                     currentProcessModulePath += hklmMachineModulePath; // += EVT.Machine
                 }
             }
+#if !CORECLR 
```

Comment:
>#if !CORECLR [](start = 0, length = 12)

We don't want to execute this code as part of inventory agent but it should be executed for all other scenarios to correct PSModulePath.